crates/holochain_keystore/src/keystore_actor.rs

```diff
@@ -81,6 +81,8 @@ impl KeystoreSenderExt for KeystoreSender {
             <Vec<u8>>::from(UnsafeBytes::from(input.data.to_vec())).into(),
         );
         async move {
+            let span = tracing::debug_span!("signing request");
```

Comment:
It's better to put the span on the await with `.instrument()` otherwise that span stays awake even when the future is pending

Reply:
What do you mean even when the future is pending? I want this to capture how long it takes for the signing request to resolve, so it seems like I want it to include the time when the future is pending, no?

Reply:
Okay. Just looked at the documentation and I see what you mean. Other futures will still be inside this span because we switch execution while still entered. Does that mean I shouldn't be trying to use tracing to measure this duration?

Reply:
You can use tracing but use the [Instrument trait](https://docs.rs/tracing-futures/0.2.5/tracing_futures/trait.Instrument.html) as this will properly show the time this future spent doing useful work. If you want to know total round trip time just put a `trace!()` before and after and you can see the timestamps when they print out.

Reply:
Thanks. I did what you suggested